src/DetailView/DetailView.jsx

```diff
@@ -126,6 +126,7 @@ export const getDetailView = (DetailComponent = Detail) => {
           {this.showAlert()}
           <DetailComponent schema={activeSchema} data={data}
             onEdit={this.handleOpenUpdateDialog} onDelete={this.handleOpenAlert}
+            url={this.props.url} id={this.props.params.id}
```

Comment:
You can get `url` and `params` at the beginning of render method where the `props` object is destructured.